server/src/services/application.service.ts

```diff
@@ -708,31 +722,51 @@ export const sendMailToApplicant = async ({
   company_recruitment_intention: string
   company_feedback: string
 }): Promise<void> => {
+  const partner = (application.caller && PARTNER_NAMES[application.caller]) ?? null
+  const jobSourceType: string = await getJobSourceType(application)
```

Comment:
```suggestion
  const jobSourceType = await getJobSourceType(application)
```
Si tu enlève le typing `string`, tu devrais avoir une inférence de typing plus précise de type `CFA | ENTREPRISE`